src/openapi.yaml

```diff
@@ -8233,7 +8239,12 @@ components:
         label:
           type: string
           description: The human-readable name for a plugin.
+        components:
+          type: array
+          items: {}
+          description: The components for which the plugin is relevant.
```

Comment:
this doesn't seem right - shouldn't this be a proper schema, my first impression is that it each item should be a string?